plugins/kafka/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+  - hmadison
```

Comment:
Hey @hmadison 

With this your are proposing yourself as a [maintainer](https://github.com/falcosecurity/evolution/blob/main/MAINTAINERS_GUIDELINES.md#onboarding-a-maintainer), not just a [reviewer](https://github.com/falcosecurity/evolution/blob/main/MAINTAINERS_GUIDELINES.md#onboarding-a-reviewer). 

Either option is acceptable, but onboarding processes and responsibilities are different. 
I'd like to check with you about the differences. and you are selecting the wanted option:

- (a) If you are proposing yourself to be a maintainer, your current change is okay. In such a case, please comment below and confirm this is what you want. I will take care of the onboarding process.
- or (b), if you are proposing yourself as a reviewer only, change `approvers` :point_right: `reviewers.`


/hold

Reply:
Sounds good! I'm after option "b". Sorry for the confusion!